easybuild/tools/environment.py

```diff
@@ -189,18 +189,39 @@ def sanitize_env():
     """
     Sanitize environment.
 
-    This function undefines all $PYTHON* environment variables,
-    since they may affect the build/install procedure of Python packages.
+    This function
 
```

Comment:
Either remove this line or change below to 'filters', 'undefines'

Reply:
fixed in 772d823